package/features

```diff
@@ -23,8 +23,8 @@ packages 'web-advanced' \
 packages 'web-advanced & autoupdater' \
 	'gluon-web-autoupdater'
 
-packages 'status-page & (mesh-batman-adv-14 | mesh-batman-adv-15)' \
+packages 'status-page & (mesh-batman-adv-legacy | mesh-batman-adv)' \
```

Comment:
I would prefer to keep the package names unchanged until we actually get rid of compat 14.